### DISCUSSION TITLE
[dropped] Добавить возможность ставить теги к строкам

### DISCUSSION THREAD

**Discussion by vb-r1id** [totumonline/totum-ru-issues-and-discussions#213] [Есть идея, а что если...]
Сделать возможность проставлять теги к строкам. Очень удобно когда нужно собрать быстрый фильтр

**Comment by ttm-alexeypolunin** [comment]
Вы же и сейчас можете в каждой строке сделать поле Селект с «тегами» и фильтровать потом по этим значениям. Я не совсем понимаю какой еще функционал сверху нужен. У нас там есть задумки на поля с автодополнением, но не уверен, что вы именно их хотите.
